selffuzz/pom.xml

```diff
@@ -0,0 +1,170 @@
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>5.9.2</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.9.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <version>5.9.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.platform</groupId>
+      <artifactId>junit-platform-commons</artifactId>
+      <version>1.9.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.platform</groupId>
+      <artifactId>junit-platform-launcher</artifactId>
+      <version>1.9.2</version>
+    </dependency>
+    <dependency>
+      <groupId>com.code-intelligence</groupId>
+      <artifactId>jazzer</artifactId>
+      <version>dev</version>
+      <scope>system</scope>
+      <systemPath>${project.basedir}/../bazel-bin/deploy/jazzer-project.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>com.code-intelligence</groupId>
+      <artifactId>jazzer-junit</artifactId>
+      <version>dev</version>
+      <scope>system</scope>
+      <systemPath>${project.basedir}/../bazel-bin/deploy/jazzer-junit-project.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>com.code-intelligence</groupId>
+      <artifactId>jazzer-api</artifactId>
+      <version>dev</version>
+      <scope>system</scope>
+      <systemPath>${project.basedir}/../bazel-bin/deploy/jazzer-api-project.jar</systemPath>
+    </dependency>
+    <!--
+    <dependency>
+      <groupId>com.code-intelligence</groupId>
+      <artifactId>jazzer-junit</artifactId>
+      <version>0.19.0</version>
+      <scope>test</scope>
+    </dependency>
+    -->
+    <dependency>
+      <groupId>com.code-intelligence.selfuzz.com.code-intelligence</groupId>
+      <artifactId>jazzer</artifactId>
+      <version>dev</version>
+      <scope>system</scope>
+      <systemPath>${project.basedir}/../bazel-bin/selffuzz/jazzer_selffuzz.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>com.google.truth</groupId>
+      <artifactId>truth</artifactId>
+      <version>1.1.3</version>
+      <scope>test</scope>
+    </dependency>
+
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.8.8</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>prepare-agent</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>report</id>
+            <phase>test</phase>
+            <goals>
+              <goal>report</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-clean-plugin</artifactId>
```

Comment:
Are the explicit definitions of the Maven plugins required? Without the default versions should be used.

Reply:
I think these came over when I copied the cifuzz example project but it's work when I delete them so apparently they're not required.